lms/djangoapps/teams/static/teams/js/views/team_card.js

```diff
@@ -47,11 +47,16 @@
         TeamCountryLanguageView = Backbone.View.extend({
             template: _.template(teamCountryLanguageTemplate),
 
+            initialize: function (options) {
+                this.country = options.countries[this.model.get('country')];
```

Comment:
Nit: this is making the assumption that the model never changes between calls to render (true in this case, but not usually true for Backbone views). I would be more likely to do the translation from the model values directly in the render method.